src/components/cylc/analysis/AnalysisTable.vue

```diff
@@ -128,50 +132,82 @@ export default {
   computed: {
     shownHeaders () {
       const times = upperFirst(this.timingOption)
-      const timingHeaders = [
-        {
-          title: `Mean T-${times}`,
-          key: `mean${times}Time`,
-          formatter: formatDuration,
-          allowZeros: false
-        },
-        {
-          title: `Std Dev T-${times}`,
-          key: `stdDev${times}Time`,
-          formatter: formatDuration,
-          allowZeros: true
-        },
-        {
-          title: `Min T-${times}`,
-          key: `min${times}Time`,
-          formatter: formatDuration,
-          allowZeros: false
-        },
-        {
-          title: `Q1 T-${times}`,
-          key: `${times.toLowerCase()}Quartiles.0`,
-          formatter: formatDuration,
-          allowZeros: false
-        },
-        {
-          title: `Median T-${times}`,
-          key: `${times.toLowerCase()}Quartiles.1`,
-          formatter: formatDuration,
-          allowZeros: false
-        },
-        {
-          title: `Q3 T-${times}`,
-          key: `${times.toLowerCase()}Quartiles.2`,
-          formatter: formatDuration,
-          allowZeros: false
-        },
-        {
-          title: `Max T-${times}`,
-          key: `max${times}Time`,
-          formatter: formatDuration,
-          allowZeros: false
+      const timingHeaders = []
+      // Check if there are any stats to show
+      let stats = false
+      for (let i = 0; i < this.tasks.length; i++) {
+        if (this.tasks[i].count > 1) {
+          stats = true
+          break
         }
-      ]
+      }
```

Comment:
```suggestion
      const stats = this.tasks.some((task) => task.count > 1)
```

Reply:
`some` is a terrible name for a method. I like my old school loops!